gdx/src/com/badlogic/gdx/utils/ReflectionPool.java

```diff
@@ -22,7 +22,12 @@
 
 /** Pool that creates new instances of a type using reflection. The type must have a zero argument constructor.
  * {@link Constructor#setAccessible(boolean)} will be used if the class and/or constructor is not visible.
+ * <p>
+ *
+ * The class is deprecated, please use {@link DefaultPool} instead.
```

Comment:
This is what the `deprecated` doclet is for, which is like `param`, `return` or `author` and has an at-sign before it. (I don't want to accidentally summon a user named author or something.)